src/ui/units/workbooks/components/Table/WorkbookEntriesTable/ChunkGroup/ChunkGroup.scss

```diff
@@ -0,0 +1,11 @@
+.dl-chunk-group {
```

Comment:
I think it's very common css class name for component inside `components/Table/WorkbookEntriesTable/ChunkGroup`